test/classes/Collector.spec.js

```diff
@@ -0,0 +1,126 @@
+import assert from 'power-assert';
+import mockgoose from 'mockgoose';
+import _ from 'lodash';
+import AutherModel from 'models/Author';
+import Collector from 'classes/Collector';
+const AuthorCollector = Collector('author');
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).